src/infection-model/rt.tsx

```diff
@@ -83,16 +83,8 @@ const getRtInputsForFacility = async (
 
   // inputs must contain no more than one record per day.
   // when we have multiples, filter out all but the most recently updated
-  // TODO: it may be possible to do this filtering directly in the query
-  // with a compound index, see #254 and replace this code if possible
   modelVersions = uniqBy(
-    orderBy(
-      modelVersions,
-      ["observedAt", "updatedAt"],
-      // uniqBy retains only the first item when it encounters duplicates,
-      // so make sure the most recently updated one is first
-      ["asc", "desc"],
-    ),
+    modelVersions,
```

Comment:
I think I would rather have the uniqueness filter move out of this function as well; it depends on the inputs being in the right order (and that order is somewhat counterintuitive), so having the two transformations be so distant from one another makes me worry this will be prone to breakage.